Common/GPU/Vulkan/VulkanQueueRunner.cpp

```diff
@@ -841,9 +841,11 @@ void VulkanQueueRunner::LogRenderPass(const VKRStep &pass, bool verbose) {
 			case VKRRenderCommand::REMOVED:
 				INFO_LOG(G3D, "  (Removed)");
 				break;
-
-			case VKRRenderCommand::BIND_PIPELINE:
```

Comment:
Should we keep this logging since we kept BIND_PIPELINE?

-[Unknown]

Reply:
That was probably a merge mistake, I'll take a look.